desktop/core/src/desktop/js/apps/notebook/app.js

```diff
@@ -953,7 +954,21 @@ huePubSub.subscribe('app.dom.loaded', app => {
         huePubSub.subscribe(
           'editor.upload.query',
           query_id => {
-            viewModel.selectedNotebook().snippets()[0].uploadQuery(query_id);
+            const useInternalHueOptimizerStorage = true;
```

Comment:
We should move this setting into the config file and have it default to false for now...